dx/utils/formatting.py

```diff
@@ -138,39 +138,11 @@ def clean_column_values_for_display(s: pd.Series) -> pd.Series:
     s = datatypes.handle_ip_address_series(s)
     s = datatypes.handle_complex_number_series(s)
 
-    s = geometry.handle_geometry_series(s)
-    s = datatypes.handle_unk_type_series(s)
-    return s
-
-
-def clean_column_values_for_hash(s: pd.Series) -> pd.Series:
-    """
-    Cleaning/conversion for values in a series to prevent
-    hash_pandas_object() errors.
-    """
-    s = geometry.handle_geometry_series(s)
-
-    s = datatypes.handle_dict_series(s)
-    s = datatypes.handle_sequence_series(s)
-    return s
-
-
-def clean_column_values_for_sqlite(s: pd.Series) -> pd.Series:
-    """
-    Cleaning/conversion for values in a series to prevent
-    errors writing to sqlite.
-    """
-    s = datatypes.handle_dtype_series(s)
-    s = datatypes.handle_interval_series(s)
-    s = datatypes.handle_complex_number_series(s)
-    s = datatypes.handle_ip_address_series(s)
-
-    s = date_time.handle_time_period_series(s)
-
     s = geometry.handle_geometry_series(s)
 
     s = datatypes.handle_dict_series(s)
     s = datatypes.handle_sequence_series(s)
+    s = datatypes.handle_unk_type_series(s)
```

Comment:
Why is the type unknown for these?

Reply:
Mostly as a catch-all for if we run into objects that aren't JSON serializable that we haven't handled in an explicit way earlier, to avoid breaking the pandas `build_json_schema()` call. Definitely open to suggestions here!